pyiron_base/server/queuestatus.py

```diff
@@ -210,7 +211,7 @@ def wait_for_job(job, interval_in_s=5, max_iterations=100):
                 )
 
 
-def wait_for_jobs(project, interval_in_s=5, max_iterations=100, recursive=True):
+def wait_for_jobs(project, interval_in_s=0, max_iterations=1, recursive=True):
```

Comment:
Is it necessary to change the defaults?

Reply:
Not really necessary, but I really think the default doesn't make much sense, because submitting a job takes like 20-30 seconds, so everything that runs in 5 seconds is multiple times faster when run locally anyway. Also at least on our cluster the HPC team is not a fan of putting high loads on login nodes by stuff like this.